### PR TITLE
feat(courses): add golangr.com

### PR DESCRIPTION
**Pull request by CleanMachine1**
## What does this PR do?
Add resource

### Description

Online course to go through some topics

### Why is this valuable (or not)?

Teaches some basic input for learning Go

### How do we know it's really free?

There is no signup

### For book lists, is it a book? For course lists, is it a course? etc.

?

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -441,6 +441,7 @@
 * [Golang in under an hour](https://www.youtube.com/watch?v=N0fIANJkwic) - Eli Goldberg
 * [Golang Tutorial - Learn Go Programming Language](https://www.geeksforgeeks.org/golang-tutorial-learn-go-programming-language/) - GeeksforGeeks
 * [Gophercises: Free Coding Exercises for Budding Gophers](https://gophercises.com) - Jon Calhoun (email address *required*)
+* [Learn Go](https://golangr.com)
```
Comment (davorpa) [inline courses/free-courses-en.md:444]: Better use full title resource and make reference to the website as author. I can't find it real name.

```suggestion
* [Learn Go \| Learn Go Programming](https://golangr.com) - golangr.com
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Complete metadata 😉 to differentiate from other resources

File: courses/free-courses-en.md (hunk 1 above)
Comment (CleanMachine1) [reply courses/free-courses-en.md:444]: Is there a mistype with the `\`?

**Review by 53jk1 — APPROVED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (davorpa) [reply courses/free-courses-en.md:444]: > Is there a mistype with the `\`?

Is the markdown escape character :wink:

Comment (eshellman) [reply courses/free-courses-en.md:444]: Makes me curious who is behind this; probably related to the book it affiliate-links to. It's not copied from anywhere else. but yes, for this "golangr.com" is a useful attribution - that's who the copyright is asserted for.

Comment (davorpa) [reply courses/free-courses-en.md]: > Makes me curious who is behind this; probably related to the book it affiliate-links to. It's not copied from anywhere else. but yes, for this "golangr.com" is a useful attribution - that's who the copyright is asserted for.

Maybe @Callebdoxley ?

**Comment by davorpa** [comment]
@calebdoxsey is yours?

**Review by davorpa — APPROVED** [review]

**Comment by calebdoxsey** [comment]
> @calebdoxsey is this tutorial website yours?

Hi @davorpa 

I own https://www.golang-book.com. I do not own https://golangr.com. Unfortunately I don't know who does and I couldn't find any attribution on their website.

**Pull request merged**